docs/instructor-guide/index.md

```diff
@@ -0,0 +1,26 @@
+# Instructor Guide
+
+This section provides detailed documentation for all aspects of using PrairieLearn as an instructor.
+
+To run and test question development locally, see the [local installation guide](../installing.md).
+
+For more details about configuring the four main aspects of your course:
+
+- [Courses](../course/index.md)
+- [Course Instances](../courseInstance.md)
+- [Assessments](../assessment/index.md)
+- [Questions](../question.md)
+
+!!! tip
+
+    For a refresher of what these aspects are, see the [PrairieLearn concepts](../concepts/index.md).
+
+Some other important links are:
+
+- [Access Control](../accessControl/index.md)
+- [Element catalogue](../elements.md)
+- [Autograders](../externalGrading.md)
+  - [Python](../python-grader/index.md)
+  - [C / C++](../c-grader/index.md)
+- [FAQ](../faq.md)
+- [syncing in-browser vs git](../sync.md)
```

Comment:
I would just use the title of the actual page, "Editing and syncing course content".

Reply:
Resolved in https://github.com/PrairieLearn/PrairieLearn/pull/11371/commits/50b8c72f43aca12c6f7f3f65dd203378ad26a914.